.gitpod.dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM rust:latest
+
+# Install mob.sh tool
+RUN curl -sL install.mob.sh | sh
+
+# Install some rustup components that are available by default.
```

Comment:
Perhaps you meant "that are **not** available by default"?

Reply:
I just realized it was me who added this >1 month ago... So here's one way, ladies and gentlemen, how a person could speak to their past self.

Reply:
Haha. yeah. These things happen with shared ownership :)

Reply:
I guess we would need to look deeper, which ones are already provided by the default rust image... but let's see first if we want to continue this path?
Adding @HenningHolmDE into the loop.